### PR TITLE
Update zktx example doc: use stable version of Rust.

### PR DESCRIPTION
**Pull request by kaikai1024**

### CODE REVIEW

File: docs/cita/system/zktx.md [hunk 1]
```diff
@@ -39,7 +39,11 @@ cd zktx_example && cargo run --release --bin gen_params
 
 #### 编译 `CITA`
 
-使用 `make debug` 或者 `make release`
+详细信息可查看 [编译 CITA] 文档，如下操作：
```
Comment (boundless-forest) [inline docs/cita/system/zktx.md:42]: Add a link?

Comment (boundless-forest) [reply docs/cita/system/zktx.md:42]: OK, I see.

File: docs/cita/system/zktx.md [hunk 2]
```diff
@@ -39,7 +39,11 @@ cd zktx_example && cargo run --release --bin gen_params
...
+
+```
```
Comment (boundless-forest) [inline docs/cita/system/zktx.md]: sh or bash?

Comment (kaikai1024) [reply docs/cita/system/zktx.md]: ok

**Review by leeyr338 — APPROVED** [review]

**Pull request merged**